notebooks/02-ingestion/12-spatial-dimensions.ipynb

```diff
@@ -0,0 +1,709 @@
+    "      \"timestampSpec\": {\n",
+    "        \"column\": \"time\",\n",
+    "        \"format\": \"iso\"\n",
+    "      },\n",
+    "      \"dimensionsSpec\": {\n",
+    "        \"dimensions\": [\n",
+    "          \"user_id\",\n",
+    "          \"first_name\",\n",
+    "          \"last_name\",\n",
+    "          \"dob\",\n",
+    "          {\n",
+    "            \"name\": \"address_lat\",\n",
+    "            \"type\": \"double\"\n",
+    "          },\n",
+    "          {\n",
+    "            \"name\": \"address_long\",\n",
+    "            \"type\": \"double\"\n",
+    "          },\n",
+    "          \"marital_status\",\n",
+    "          {\n",
+    "            \"name\": \"income\",\n",
+    "            \"type\": \"double\"\n",
+    "          },\n",
+    "          \"signup_ts\"\n",
+    "        ],\n",
+    "        \"spatialDimensions\": [\n",
+    "          {\n",
+    "             \"dimName\": \"address_spatial\",\n",
+    "             \"dims\": [\n",
+    "               \"address_lat\",\n",
+    "               \"address_long\"\n",
+    "             ]\n",
+    "          }\n",
+    "        ]\n",
+    "      }\n",
+    "    },\n",
+    "    \"tuningConfig\": {\n",
+    "      \"type\": \"index_parallel\",\n",
+    "      \"partitionsSpec\": {\n",
+    "        \"type\": \"dynamic\"\n",
+    "      }\n",
+    "    }\n",
+    "  }\n",
+    "}\n",
+    "headers = {\n",
+    "  'Content-Type': 'application/json'\n",
+    "}\n",
+    "\n",
+    "task = rest_client.post(\"/druid/indexer/v1/task\", json.dumps(spatial_index_spec), headers=headers)\n",
+    "task_id = json.loads(task.text)['task']\n",
+    "wait_for_task(task_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6bd176f7-c443-41ec-af7a-1e50d69fae01",
+   "metadata": {},
+   "source": [
+    "## Filter spatial data \n",
+    "Querying spatial data also requires the using indexed filtering currently requires the use of the native query API. \n",
+    "Spatial filters are specified as follows:\n",
+    "\n",
+    "```\n",
+    "\"filter\": {\n",
+    "  \"type\": \"spatial\",\n",
+    "  \"dimension\": <name_of_spatial_dimension>,\n",
+    "  \"bound\": <bound_type>\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "There are 3 types of bounds to find spatial points:\n",
+    "- rectangular - matches any point that falls within the specified rectangle \n",
+    "- radius - matches any point that falls within the circle defined by center coordinates and a radius \n",
+    "- polygon - matches any point within the area of the polygon as defined by a set of points\n",
+    "\n",
+    "\n",
+    "### Query with spatial data\n",
+    "The following TopN query will aggregate minimum and maximum income among the users found in the filtered geographical area and also count them.\n",
+    "The query is the same across all three examples, each example uses a different spatial filter bound type: \"rectangular\",\"radius\" and \"polygon\".\n",
```

Comment:
Can we use code font instead of quotation marks? Or omit quotation marks?

Reply:
I'll add the code font. I try to use the columns names in quotes because that is a best practice to avoid future SQL keyword collisions.

Reply:
done